src/containers/AIEvals/AIEvaluationList/AIEvaluationList.module.css

```
@@ -128,3 +148,31 @@
+.SubInfoIcon {
+  height: 14px !important;
+  color: #2c2d2f;
+  flex-shrink: 0;
+}
+
+.SubInfoLink {
+  font-size: 13px;
+  color: #119656;
+  text-decoration: underline;
+  text-underline-offset: 2px;
+  word-break: break-word;
```

Stylelint (17.10.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)